.github/workflows/release.yml

```diff
@@ -125,6 +125,10 @@ jobs:
       with:
         path: /tmp/binaries
 
+    - name: Calculate sha256 hashes
```

Comment:
Why not put them into a separated filed that is uploaded later as an artifact? 

options: 
- for each binary file a new txt that the name of the file is the sha and a new artifact
- for each binary file add the sha to a txt with the format: `binay-name:sha` or similar and upload that binary later as a new artifact

I particularly don't like the idea of attach to the checksum the shasum

Reply:
Maybe we can use something like this https://github.com/marketplace/actions/generate-checksum